.gitmodules

```diff
@@ -12,6 +12,10 @@
 [submodule "submodules/clog"]
 	path = submodules/clog
 	url = https://github.com/microsoft/CLOG.git
+[submodule "submodules/xdp-tools"]
+	path = submodules/xdp-tools
+	url = https://github.com/xdp-project/xdp-tools.git
+	branch = v1.4.0
```

Comment:
Looks like the latest is `v1.4.2`. Should we be using that instead?

Reply:
OK now experimenting v1.4.2. v1.4.0 was latest when I made this PR originally